package.json

```diff
@@ -13,7 +13,7 @@
     "test": "npm-run-all --parallel test:unit test:cypress",
     "test:unit": "kcd-scripts test --no-watch",
     "test:unit:watch": "kcd-scripts test",
-    "test:cypress:serve": "serve --clipless --local --port 13370 ./cypress/fixtures/test-app",
+    "test:cypress:serve": "serve -l 13370 ./cypress/fixtures/test-app",
```

Comment:
Why change to shorthand? Has serve stopped supporting full nsmes for options?

Reply:
You're right, oversight on my part. Long version would be `--listen`

They did change their API quite drastically. Since it's missing from their docs, here's the help for easy reference.
```
  serve - Static file serving and directory listing

  USAGE
      $ serve --help
      $ serve --version
      $ serve [-l listen_uri [-l ...]] [directory]

      By default, serve will listen on 0.0.0.0:3000 and serve the
      current working directory on that address.

      Specifying a single --listen argument will overwrite the default, not supplement it.

  OPTIONS
      --help                              Shows this help message
      -v, --version                       Displays the current version of serve
      -l, --listen listen_uri             Specify a URI endpoint on which to listen (see below) -
                                          more than one may be specified to listen in multiple places
      -d, --debug                         Show debugging information
      -s, --single                        Rewrite all not-found requests to `index.html`

  ENDPOINTS
      Listen endpoints (specified by the --listen or -l options above) instruct serve
      to listen on one or more interfaces/ports, UNIX domain sockets, or Windows named pipes.

      For TCP ports on hostname "localhost":
          $ serve -l 1234
      For TCP (traditional host/port) endpoints:
          $ serve -l tcp://hostname:1234
      For UNIX domain socket endpoints:
          $ serve -l unix:/path/to/socket.sock
      For Windows named pipe endpoints:
          $ serve -l pipe:\\.\pipe\PipeName
```